script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_9.txt

```diff
@@ -1,20 +1,20 @@
 <Page1>
 I sit face to face with Akiha at the dinner table.
-Hisui and Kohaku are focused on serving us.
+Hisui and Kohaku-san are focused on serving us.
 A master and their servants do not sit at the same table.
 ...Naturally, it is an uncomfortable dinner for me, who is used to the company of the Arima family.
 <Page2>
 In addition, I have completely forgotten about table manners.
 Humans are experts in optimization, relegating unused memories to a corner of the mind. In other words, there is no reason for me to remember table manners.
 <Page3>
-It is quite a tense but also thrilling experience to see Akiha's eyebrows raise at my every move.
+It is quite a tense, yet thrilling experience to see Akiha's eyebrows rise at every move I make.
```

Comment:
I think `raise` is still correct here. A non-scientific survey of the literature seems to agree

![2021-10-16-212007_698x284_scrot](https://user-images.githubusercontent.com/2476799/137606549-b16b8f2f-e19f-4edf-b60e-7a9994e4981b.png)

![2021-10-16-212013_653x288_scrot](https://user-images.githubusercontent.com/2476799/137606548-2640793f-c7d9-48bc-bffe-11a6507e12b2.png)

Reply:
Yeah I think you're right.